docs/source/multi_gpu.rst

```diff
@@ -611,38 +611,71 @@ This is useful when dealing with large Transformer based models, or in environme
 
 Lightning currently offers the following methods to leverage model parallelism:
 
-- Optimizer Sharded Training (partitioning your gradients and optimizer state across multiple GPUs, for reduced memory overhead)
+- Sharded Training (partitioning your gradients and optimizer state across multiple GPUs, for reduced memory overhead with **no performance loss**)
 
-Optimizer Sharded Training
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-Lightning integration of optimizer sharded training provided by `Fairscale <https://github.com/facebookresearch/fairscale>`_.
+Sharded Training
```

Comment:
Would you mind adding some images and maybe even put a link to your blogpost when released.

Reply:
Absolutely, will do in a followup PR after release :)